apps/admin/frontend/src/screens/write_ins_screen.test.tsx

```diff
@@ -144,147 +140,165 @@ test('ballot pagination', async () => {
 });
 
 test('adjudication', async () => {
-  apiMock.expectGetWriteIns(mockWriteInRecords);
+  const contestId = 'zoo-council-mammal';
+  apiMock.expectGetWriteInSummary([mockWriteInSummaryPending(contestId, 3)]);
   apiMock.expectGetCastVoteRecordFiles([]);
 
   renderInAppContext(<WriteInsScreen />, {
     electionDefinition,
     apiMock,
   });
 
-  // transcribe
+  // open adjudication screen
+  const mockWriteInRecords = ['0', '1'].map(mockWriteInRecordPending);
+  apiMock.expectGetWriteIns(mockWriteInRecords, contestId);
+  const mockWriteInCandidate: WriteInCandidateRecord = {
+    id: 'lemur',
+    name: 'Lemur',
+    contestId: 'zoo-council-mammal',
+    electionId: 'id',
+  };
+  apiMock.expectGetWriteInCandidates([mockWriteInCandidate], contestId);
   apiMock.expectGetWriteInImageView(mockWriteInRecords[0].id);
   apiMock.expectGetWriteInImageView(mockWriteInRecords[1].id); // prefetch
-  userEvent.click(await screen.findByText('Transcribe 3'));
-  userEvent.type(
-    await screen.findByPlaceholderText('transcribed write-in'),
-    'Dark Helmet'
-  );
-  apiMock.apiClient.transcribeWriteIn
+  userEvent.click(await screen.findByText('Adjudicate 3'));
+
+  await screen.findButton('Zebra');
+  await screen.findButton('Lion');
```

Comment:
nit: Are we able to use `getButton()` calls instead once we've waited for the first one to get loaded? Or do they render at different times? (The await overhead probably isn't much, but just as a pattern for when we expect things to already be in the DOM)

Reply:
yes, we could use `getButton` here. I didn't because I didn't like the asymmetry of `find`ing one and `get`ing another, but that was more aesthetic lol. the buttons are equivalent to each other in when they appear on the screen, so i think the code is slightly confusing that it might indicate some button is unique or central.

i'll await some header and then `get` all the buttons as a solution.